cli/command/image/list.go

```diff
@@ -75,6 +81,26 @@ func runImages(ctx context.Context, dockerCLI command.Cli, options imagesOptions
 		filters.Add("reference", options.matchName)
 	}
 
+	if options.tree {
+		if options.quiet {
+			return errors.New("--quiet is not yet supported with --tree")
+		}
+		if options.noTrunc {
+			return errors.New("--no-trunc is not yet supported with --tree")
+		}
+		if options.showDigests {
+			return errors.New("--show-digest is not yet supported with --tree")
+		}
+		if options.format != "" {
+			return errors.New("--format is not yet supported with --tree")
+		}
```

Comment:
Also slightly wondering if we should use `--format=tree` for this (although `--tree` could be a "porcelain" option for setting that) 🤔

Reply:
`tree` being a `format` would mean taking away the option to specify custom columns though.
Not sure if we want that?

Reply:
Depends a bit on how it's implemented. For the default (`table`) output, we have `--format=table` to pick the default table output, but it takes arguments, so can be used to produce a custom table;

```bash
docker image ls --format=table
REPOSITORY       TAG               IMAGE ID       CREATED         SIZE
docker-cli-dev   latest            88c125e27aee   21 hours ago    749MB
docker-dev       latest            60f88d03ca84   4 days ago      2.21GB
golang           1.22              2bd56f00ff47   6 days ago      1.21GB

docker image ls --format "table {{.Tag}}\t{{.ID}}"
TAG               IMAGE ID
latest            88c125e27aee
latest            60f88d03ca84
1.22              2bd56f00ff47
```